renderer/reducers/modal.js

```diff
@@ -139,6 +168,14 @@ const ACTION_HANDLERS = {
   [SET_MODALS]: (state, { modals }) => {
     state.modals = modals
   },
+
+  [OPEN_DIALOG]: (state, { id }) => {
```

Comment:
Lets add reducer snapshot tests for these new actions

Reply:
added